pkg/preparer/orchestrate.go

```diff
@@ -150,22 +151,28 @@ func (p *Preparer) handlePods(podChan <-chan pods.Manifest, quit <-chan struct{}
 			if !working {
 				p.tryRunHooks(hooks.AFTER_AUTH_FAIL, pods.NewPod(manifestToLaunch.ID(), pods.PodPath(p.podRoot, manifestToLaunch.ID())), &manifestToLaunch, manifestLogger)
 			}
-		case <-time.After(1 * time.Second):
```

Comment:
I think we should always be pausing here. There are lots of cases where the work done inside the `working` block will terminate quickly in case of errors.